src/worker/tests/mod.rs

```diff
@@ -84,6 +84,10 @@ mod revs {
         mkid("mnkoropy", "5f4dc18dd03158f9ec72e528ffaca2f4c73d3b4b")
     }
 
+    pub fn immutable_bookmark() -> RevId {
```

Comment:
is this necessary, or could we achieve the same thing by moving `main`? if there's some simplification or test feature that it enables, that's fine - i just want to check that there's a purpose to the repo update